test/Tests.Infrastructure/RavenTestBase.cs

```diff
@@ -220,6 +223,8 @@ protected virtual DocumentStore GetDocumentStore(Options options = null, [Caller
 
                     store.BeforeDispose += (sender, args) =>
                     {
+                        CheckForMissingAttachmentsAndThrowIfNeeded(store, name, serverToUse, caller);
```

Comment:
do it only for tests that have RavenTestCategory.Replication and Attachments?

Reply:
I would leave it for all test until we actually fix the issue, or at least will have Category on most of the tests

Reply:
Let's talk about this, ping me tomorrow